components/bl-pdfViewer-component/src/subcomponents.js

```diff
@@ -21,7 +23,31 @@ export function NoData() {
 }
 
 export function Controls(props) {
-  const { pageIndex, setPageIndex, inputRef, handlerPageChange, numPages, display, controlsRef } = props;
+  const { pageIndex, setPageIndex, inputRef, numPages, display, controlsRef } = props;
+
+  const [inputValue, setInputValue] = useState(1);
+
+  const handlerSubmit = () => {
+    if (Number(inputValue)) {
+      const page = ensureRange(inputValue, { min: 1, max: numPages });
+
+      setPageIndex(page);
+      setInputValue(page);
+    } else {
+      setPageIndex(1);
+      setInputValue(1);
+    }
+  };
```

Comment:
add useCallback

rename handlerSubmit -> onSubmit

Reply:
I'm not sure what is needed use useCallback on native HTML elements.
https://github.com/facebook/react/issues/17055